src/main/scala/io/getclump/Tuples.scala

```diff
@@ -49,4 +49,19 @@ protected[getclump] trait Tuples {
   protected[getclump] def fetch4[A, B, C, D, E, F](fetch: (A, B, C, D, E) => Future[Iterable[F]]) = (params: (A, B, C, D), values: E) => (params, values) match {
     case ((a, b, c, d), e) => fetch(a, b, c, d, e)
   }
+
+  protected[getclump] def adapt[K, V](fetch: K => Future[V]) = (keys: Set[K]) =>
```

Comment:
True. Updated to use Iterable and to use the A B C D E type naming of the other methods in this class